src/admin-portal/create-campaign/Technologies.jsx

```diff
@@ -13,13 +13,13 @@ import { useCampaignContext } from "../../hooks/use-campaign-context";
 import { useBubblyBalloons } from "../../lib/bubbly-balloon/use-bubbly-balloons";
 import { updateCampaignTechnologies } from "../../requests/campaign-requests";
 
-function Technology ({ tech, handleRemove }) {
+function Technology ({ tech, handleRemove, navigate }) {
```

Comment:
Good job bruv!
Please remove use the hook.
Hooks are called at every render of a component